querydsl-jdo/src/main/java/com/querydsl/jdo/JDOQLSerializer.java

```diff
@@ -105,7 +105,7 @@ public void serialize(QueryMetadata metadata, boolean forCountRow, boolean subQu
         final Predicate having = metadata.getHaving();
         final List<OrderSpecifier<?>> orderBy = metadata.getOrderBy();
 
-        constantToLabel.push(new HashMap<Object,String>());
+        //constantToLabel.push(new HashMap<Object,String>());
```

Comment:
No. I will clean it up later.
On 2 Apr 2015 05:53, "John Tims" <notifications@github.com> wrote:

> In querydsl-jdo/src/main/java/com/querydsl/jdo/JDOQLSerializer.java
> <https://github.com/querydsl/querydsl/pull/1298#discussion_r27629680>:
>
> > @@ -105,7 +105,7 @@ public void serialize(QueryMetadata metadata, boolean forCountRow, boolean subQu
> >          final Predicate having = metadata.getHaving();
> >          final List<OrderSpecifier<?>> orderBy = metadata.getOrderBy();
> >
> > -        constantToLabel.push(new HashMap<Object,String>());
> > +        //constantToLabel.push(new HashMap<Object,String>());
>
> Any reason to keep these lines around as comments?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/querydsl/querydsl/pull/1298/files#r27629680>.
>